app/patients/reports/controller.js

```diff
@@ -276,6 +276,8 @@ export default AbstractReportController.extend(PatientDiagnosis, PatientVisits,
 
   isStatusReport: computed('reportType', function() {
     let reportType = this.get('reportType');
+    this.set('startDate', null);
+    this.set('endDate', null);
```

Comment:
With this, it erases the dates every time you change the Report Type. Although it's funny, they still look like they're filled in in the text inputs. But the actual properties are erased, as if you set the dates on Admissions Detail, switch to Admissions Summary and then click Generate Report and it'll say your start date is missing.

You could just make it conditional so it only sets these if reportType === 'status'.

Reply:
the latest commit should fix that issue.

Reply:
cool, thanks for making all those revisions but I think we fixed like 3 issues here, so good job!